index.bs

```diff
@@ -5920,16 +5920,16 @@ implement, as they determine the characteristics of the different
 filter types. They are inspired by formulas found in the
 <a href="https://www.w3.org/2011/audio/audio-eq-cookbook.html">Audio EQ Cookbook</a>.
 
-The transfer function for the filters implemented by the
-{{BiquadFilterNode}} is:
+The {{BiquadFilterNode}} processes audio with a time-domain equation of:
 
 <pre nohighlight>
 $$
-H(z) = \frac{\frac{b_0}{a_0} + \frac{b_1}{a_0}z^{-1} + \frac{b_2}{a_0}z^{-2}}
-						{1+\frac{a_1}{a_0}z^{-1}+\frac{a_2}{a_0}z^{-2}}
+y(n) + \frac{a_1}{a_0} y(n-1) + \frac{a_2}{a_0} y(n-2) =
+	\frac{b_0}{a_0} x(n) + \frac{b_1}{a_0} x(n-1) + \frac{b_2}{a_0} x(n-2)
 $$
 </pre>
 
+
 The initial filter state is 0.
 The coefficients in the transfer function above are different for
```

Comment:
My mistake.  We do reference the transfer function here.  Maybe best to leave the transfer function in?  (Sorry about that!)